### DISCUSSION TITLE
uLaunchElf

### DISCUSSION THREAD

**Discussion by iWebbIO** [israpps/PlayStation2-Basic-BootLoader#43] [Q&A]
Dude i beg you
Tell me how to open ulaunchelf with ps2bbl
Is there even a way!?

**Comment by israpps** [comment]
> Dude i beg you Tell me how to open ulaunchelf with ps2bbl Is there even a way!?

yes. of course....

Maybe if you took a second to read the config file... you would find out by yourself.

if you didnt touch the stock config, PS2BBL will boot wLaunchELF if you press R1 on the logo screen
